scrowl-io/owl-ui
# [sc-363] - Install OSG brand webfonts

### Short summary
Installing the OSG brand font family ([Inter Typeface](https://stage.osg.ca/brand/#fonts)) will ship it with the app and guarantee that even without an internet connection, the users will have access to the brand font.

---

### Test steps
1. Run `yarn start:storybook` and checkout the button font
2. Add a new `<p>` element with some text and then change its font family to `font-family: 'Inter'` in the scss file. Ensure this font family shows up in storybook.
